app/views/layouts/layout.ejs

```diff
@@ -30,6 +30,7 @@
             <% if (currentUser) { %>
             <li><a href="/users">Participants</a></li>
             <li><a href="/teams">Teams</a></li>
+            <li><a href="https://github.com/CodingForProduct/cfp_website/wiki">Wiki</a></li>
```

Comment:
@jendiamond Can you make the link visible to everyone instead of logged in users? I don't see a reason to keep it hidden.